src/components/filters/MeasureValueFilter/OperatorDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+// (C) 2007-2019 GoodData Corporation
+import * as React from "react";
+import { injectIntl, InjectedIntlProps } from "react-intl";
+import classNames from "classnames";
+import noop from "lodash/noop";
```

Comment:
This lodash import is inconsistent with how we import lodash in other places (even one line below). Please use the `import x = require("lodash/x")` syntax.